deps/prebuilt/include/TAEF/WexTestClass.h

```diff
@@ -784,14 +864,54 @@ namespace WEX
 ///          TEST_CLASS(TestFeatureClass)
 ///      }
 /// \endcode
+#if defined (__clang__)
 #define TEST_CLASS(className) \
+protected: \
+    static constexpr auto c_taefClassName = L# className; \
```

Comment:
I agree. I was having a few issues using the Array version to initialize wchar_t* types in the structs but I didn't put much time in. Probably just need a &array[0] or to muck with the consts to get it to be happy.

---
In reply to: [92519559](https://github.com/Microsoft/WinObjC/pull/1523#discussion_r92519559) [](ancestors = 92519559)